### PR TITLE
add(examples): with-indent

### PR DESCRIPTION
**Pull request by fouad**

### CODE REVIEW

**Comment by timneutkens** [comment]
Seems that the library has gotten quite large 🤔 https://bundlephobia.com/result?p=@indent/audit@0.0.20

**Comment by fouad** [comment]
Thanks for pointing that out! Fixed in `v0.0.21` https://bundlephobia.com/result?p=@indent/audit@0.0.21

**Review by timneutkens — CHANGES_REQUESTED** [review]
Could you update the readme? We changed the structure a bit

**Comment by fouad** [comment]
@timneutkens yup, done!

File: examples/with-indent/pages/find.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import { audit } from '@indent/audit'
+import Page from '../components/Page'
+
+export default class extends Component {
+  state = { sessionId: '' }
+
+  static getInitialProps = req => {
+    audit.write({
+      event: 'server_side_event',
```
Comment (lfades) [inline examples/with-indent/pages/find.js]: If you want to showcase server side events let's use API Routes, but not an empty `getInitialProps` like in this case.

File: examples/with-indent/pages/find.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import { audit } from '@indent/audit'
+import Page from '../components/Page'
+
+export default class extends Component {
```
Comment (lfades) [inline examples/with-indent/pages/find.js]: Let's do a functional component with Hooks here.

File: examples/with-indent/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```
Comment (lfades) [inline examples/with-indent/pages/index.js]: Remove all react imports unless `React` is explicitly used.

File: examples/with-indent/pages/_app.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import App from 'next/app'
+import Router from 'next/router'
+import audit from '@indent/audit'
+
+audit.init({
+  // Find your DSN on Indent Docs https://indent.com/docs/quickstart
+  dsn: '{{INDENT_INPUT_DSN}}',
```
Comment (lfades) [inline examples/with-indent/pages/_app.js]: If this variable is required for the example to work properly, add a `now.json` that requires it as a secret, so when trying to deploy using the deployment button in the example, ZEIT Now will ask for the secret before making the deployment.

File: examples/with-indent/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+
+Deploy it to the cloud with [ZEIT Now](https://zeit.co/import?filter=next.js&utm_source=github&utm_medium=readme&utm_campaign=next-example) ([Documentation](https://nextjs.org/docs/deployment)).
+
+
+## About this example
+
+This example shows how to use [Next.js](https://github.com/zeit/next.js) with [Indent](https://indent.com/?ref=nextjs).
+
+Follow the [Indent Quickstart](https://indent.com/docs/quickstart) for step by step instructions.
```
Comment (lfades) [inline examples/with-indent/README.md]:
```suggestion
```

File: examples/with-indent/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Example app with Indent
+
```
Comment (lfades) [inline examples/with-indent/README.md]:
```suggestion

This example shows how to use [Next.js](https://github.com/zeit/next.js) with [Indent](https://indent.com/?ref=nextjs).

Follow the [Indent Quickstart](https://indent.com/docs/quickstart) for step by step instructions.

```

File: examples/with-indent/README.md (hunk 5 above)
Comment (lfades) [inline examples/with-indent/README.md]:
```suggestion
```

**Review by lfades — CHANGES_REQUESTED** [review]
Requested some changes in my suggestions, also please run the linter over the files :pray:

**Comment by timneutkens** [comment]
Hey @fouad could you update the PR 🙏 Then we'll re-open

**Pull request closed without merging**